pom.xml

```diff
@@ -2491,6 +2491,13 @@
             <type>test-jar</type>
             <scope>test</scope>
          </dependency>
+          <dependency>
```

Comment:
This is exactly why I stuck to the 3-space indent in all the POMs where I copied the `component-processor` dependency ;)